src/Components/Publishing/Display/__tests__/DisplayCanvas.test.tsx

```diff
@@ -11,38 +12,73 @@ import { CanvasVideo } from "../Canvas/CanvasVideo"
 
 import {
   Campaign,
+  StandardArticleHostedAdCanvas,
   UnitCanvasImage,
   UnitCanvasOverlay,
   UnitCanvasSlideshow,
   UnitCanvasTracked,
   UnitCanvasVideo,
 } from "../../Fixtures/Components"
 
+jest.mock("sharify", () => ({
+  data: {
+    HASHTAG_LAB_ADS_ALLOWLIST: "alloweduser@email.com",
```

Comment:
As one part of the test, might want to pass in multiple users

Reply:
Good point!